docs/the_black_code_style/future_style.md

```diff
@@ -122,33 +121,6 @@ exceed the line length limit. Line continuation backslashes are converted into
 parenthesized strings. Unnecessary parentheses are stripped. The stability and status of
 this feature istracked in [this issue](https://github.com/psf/black/issues/2188).
 
-(labels/wrap-long-dict-values)=
```

Comment:
I don't think this should be removed. We happened to have these longer descriptions only for unstable features, but I think that was just by chance.